pkg/grpc/server.go

```diff
@@ -147,6 +146,14 @@ func NewServersFromConfigurationAndServe(configurations []*configuration.ServerC
 			}))
 		}
 
+		if len(configuration.Relays) != 0 {
+			handler, err := newStreamRoutingFromConfiguration(configuration.Relays, grpcClientFactory, group)
+			if err != nil {
+				return util.StatusWrap(err, "Failed to create authenticator RPC client")
```

Comment:
Wrong error description? Do we even need one, given that the implementation already wraps all errors?